.travis.yml

```diff
@@ -42,6 +38,7 @@ before_script:
   # Decodes to e.g. `export VARIABLE=VALUE`
   - $(base64 --decode <<< "ZXhwb3J0IFNBVUNFX0FDQ0VTU19LRVk9YTAzNTM0M2YtZTkyMi00MGIzLWFhM2MtMDZiM2VhNjM1YzQ4")
   - $(base64 --decode <<< "ZXhwb3J0IFNBVUNFX1VTRVJOQU1FPXJ1YnlvbnJhaWxz")
+  - if [[ $GEM = *as*  ]] ; then sudo apt-get update && sudo apt-get -y install ffmpeg mupdf mupdf-tools ; fi
```

Comment:
Let’s check for `[[ $GEM = *ast* ]]`. The `GEM=as PRESERVE_TIMEZONES=1` run doesn’t need ffmpeg or mupdf.